db/seeds.rb

```diff
@@ -11,8 +11,7 @@
 employee_4 = Employee.create(email: "czworka@gmail.com", password: "444444")
 employee_5 = Employee.create(email: "piatka@gmail.com", password: "555555")
 
-kudo1 = Kudo.create! :title => Faker::Beer.style, :content => Faker::Coffee.notes, :giver_id => 1, :receiver_id => 5
-kudo2 = Kudo.create! :title => Faker::Beer.style, :content => Faker::Coffee.notes, :giver_id => 2, :receiver_id => 4
-kudo3 = Kudo.create! :title => Faker::Beer.style, :content => Faker::Coffee.notes, :giver_id => 3, :receiver_id => 3
-kudo4 = Kudo.create! :title => Faker::Beer.style, :content => Faker::Coffee.notes, :giver_id => 4, :receiver_id => 2
-kudo5 = Kudo.create! :title => Faker::Beer.style, :content => Faker::Coffee.notes, :giver_id => 5, :receiver_id => 1
\ No newline at end of file
+1.upto(4) do |i|
+x = i + 1
+  kudo = Kudo.create(title: Faker::Beer.style, content: Faker::Coffee.notes, giver_id: "#{i}", receiver_id: "#{x}")
```

Comment:
I think `receiver_id: #{i+1}` would be easier to read, there is no need to introduce another variable, which is used only in one place

Reply:
in fact, it is better. I did not think about it :)